src/stylesheets/main.scss

```diff
@@ -427,3 +429,21 @@ pre .language-plaintext {
     font-weight: normal;
   }
 }
+
+// Extra DSD24 styles
+
+.app-prose-scope .dsd24-lead-paragraph {
```

Comment:
I know this isn't exactly adding much to the CSS file size, but if we're just including this font size and the below margin, could we just use override classes on the page itself? (ie: `govuk-!-font-size-24` and `govuk-margin-bottom-9`)

Reply:
Have used the override classes now :)